app/(main)/(campaigns)/_layout.tsx

```diff
@@ -1,40 +1,55 @@
-import { DrawerToggle } from '@/components/ui';
-import { useAuthContext } from '@/contexts';
-import { useBreakPoints } from '@/hooks';
-import { Stack, useRouter } from 'expo-router';
-import { useEffect } from 'react';
+import { DrawerToggle } from "@/components/ui";
+import { useAuthContext } from "@/contexts";
+import { useBreakPoints } from "@/hooks";
+import { Stack, useRouter } from "expo-router";
+import { useEffect } from "react";
+import { useNavigation } from "@react-navigation/native";
+import { IconButton } from "react-native-paper";
+
+const GoBackButton = () => {
```

Comment:
Move the back button to a common component so that it can be used in other places too